apps/discovery_streams/mix.exs

```diff
@@ -0,0 +1,78 @@
+defmodule DiscoveryStreams.Mixfile do
+  use Mix.Project
+
+  def project do
+    [
+      app: :discovery_streams,
+      version: "2.2.0",
+      build_path: "../../_build",
+      config_path: "../../config/config.exs",
+      deps_path: "../../deps",
+      lockfile: "../../mix.lock",
+      elixir: "~> 1.8",
+      elixirc_paths: elixirc_paths(Mix.env()),
+      test_paths: test_paths(Mix.env()),
+      compilers: [:phoenix] ++ Mix.compilers(),
+      start_permanent: Mix.env() == :prod,
+      deps: deps(),
+      aliases: aliases(),
+    ]
+  end
+
+  def application do
+    [
+      mod: {DiscoveryStreams.Application, []},
+      extra_applications: [:prometheus_plugs, :prometheus_phoenix, :logger, :runtime_tools]
+    ]
+  end
+
+  defp elixirc_paths(env) when env in [:test, :integration], do: ["lib", "test/support"]
+  defp elixirc_paths(_), do: ["lib"]
+
+  defp test_paths(:integration), do: ["test/integration"]
+  defp test_paths(_), do: ["test/unit"]
+
+  defp deps do
+    [
+      {:brook, "~> 0.3"},
```

Comment:
you might want to update this brook reference to `0.4.0` since the top-level mixfile is floating brook to the most recent `0.4.9` and this pre-1.0 reference i think translates to the latest minor version (why you haven't had deps.get blowing up on this i expect).
probably a nit-pick but it'll make it clearer to anyone coming in to _just_ modify discovery streams what version of brook they're dealing with.